demo/Meter.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<title>Meter Styling</title>
+		<link rel="stylesheet" href="../dist/css/chassis.css">
```

Comment:
your _Meter.scss file isn't even being included in the compiled chassis.css code.  Did you open this file when testing locally?